cmd/root.go

```diff
@@ -66,16 +68,15 @@ func Execute() {
 }
 
 func initFuncs(cmd *cobra.Command, args []string) {
-	database(cmd, args)
-
-	logLvlErr := logLevel(cmd, args)
+	database()
+	logLvlErr := logLevel()
 	if logLvlErr != nil {
 		log.Fatal("Could not set log level: ", logLvlErr)
 	}
 
 }
 
-func database(cmd *cobra.Command, args []string) {
+func database() {
```

Comment:
I wonder if we should rename this function? Seems like it's assigning config values to variables in the code, and handling more than the db

Reply:
yah, that's not the most descriptive name